### PR TITLE
upgrade documentation

### PR DESCRIPTION
**Pull request by DStorck**
updated k2 documentation to include instructions to upgrade master nodes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -409,6 +409,21 @@ docker run $K2OPTS quay.io/samsung_cnct/k2:latest ./up.sh --config $HOME/.kraken
 
 Replace `foo.yaml` with the name of the configuration file you intend to use.
 
+#### Upgrading Kubernetes Version of Master Nodes
+As mentioned above, before you can upgrade the Kubernetes version, you will first need to update your configuration file with the intended version and run:
+
+```bash
+docker run $K2OPTS quay.io/samsung_cnct/k2:latest ./up.sh --config $HOME/.kraken/<your_config>.yaml
```
Comment (coffeepac) [inline README.md]: this needs a warning that if you have updated other options that terraform will run wild.  ie:  if you're using the default config for AWS and ContainerLinux has released a new version of CoreOS then your etcd nodes will get trashed and a change will be staged for every node in the cluster that has an ASG.

On a related node, I think we should build additional guards around this process so we don't accidentally torch things.

File: README.md [hunk 2]
```diff
@@ -409,6 +409,21 @@ docker run $K2OPTS quay.io/samsung_cnct/k2:latest ./up.sh --config $HOME/.kraken
...
+```
+
+Then run:
+
+ ```bash
+docker run $K2OPTS quay.io/samsung_cnct/k2:latest ./upgrade.sh --config $HOME/.kraken/<your_config>.yaml
```
Comment (coffeepac) [inline README.md]: does upgrade.sh only operate on the master nodes right now?

Comment (DStorck) [inline README.md]: yes. upgrade tears down the master nodes one by one and leaves the cluster functional during the process, but doesn't touch the kublelets

File: README.md (hunk 1 above)
Comment (joejulian) [inline README.md]: > On a related node, I think we should build additional guards around this process so we don't accidentally torch things.

As part of this PR? If not, file an issue please.

Comment (coffeepac) [inline README.md]: additional guards should not be in this PR.  the warning should be.

There is a lineitem in the k2 v1.0 tracking ticket for protecting the etcd nodes, which are the ones that matter for now as we don't have plans for how to upgrade them.

File: README.md (hunk 2 above)
Comment (coffeepac) [inline README.md]: okay.  solid.

File: README.md (hunk 1 above)
Comment (DStorck) [inline README.md]: @coffeepac I'm not sure that terraform is being touched at all with upgrade.sh . Are you sure that etcd nodes would get touched in this process?  Upgrade.sh uses a module to delete master nodes specifically and then delete them off AWS. They are recreated by the ASG

Comment (DStorck) [inline README.md]: @coffee are you concerned that terraform would run wild on up.sh?

Comment (coffeepac) [inline README.md]: @DStorck yes, the issues come when calling `cluster up`.  that command does a `terraform apply` and that command will do a replace of etcd nodes if the OS version changed.  note:  This is how I destroyed the first common-tools production cluster.

Comment (joejulian) [inline README.md]: Ah, I see the confusion. Should line 416 be up.sh or upgrade.sh?

Comment (DStorck) [inline README.md]: Okay so this warning should have been there in the first place with the mentioned of running up again, it is not specific to upgrade.

Comment (DStorck) [inline README.md]: @coffeepac There is already a warning section right above these lines starting around line 387. It mentions not changing etcd and specifically says what can be modified. Are there other things I should put in here?

**Pull request merged**